corehq/apps/case_importer/do_import.py

```diff
@@ -277,7 +277,7 @@ def get_create_caseblock(self):
     def get_update_caseblock(self):
         extras = self._get_caseblock_kwargs()
         if self.uploaded_owner_id or self.uploaded_owner_name:
-            extras['owner_id'] = self._get_owner_id(),
+            extras['owner_id'] = self._get_owner_id()
```

Comment:
That can be hard to spot. Is there a way to get Stickler to insist that we write tuples with brackets? (A quick google revealed nothing. If you aren't immediately sure, and when I have some spare time, I'll dig a little deeper. It's just that this is the second issue in a week related to a single comma.)

Reply:
Agreed!  I can't think of a situation where it's a good idea to declare a tuple that way, nor of a time when that was someone's intent.  Looks like pylint does have a checker called `trailing-comma-tuple`, though I think it's python3 only:

![image](https://user-images.githubusercontent.com/2367539/58339414-932c1c80-7e17-11e9-9b84-ed007a928e09.png)

(I've been using python2 still, so I didn't see it)